lightning/src/ln/features.rs

```diff
@@ -415,6 +417,9 @@ mod sealed {
 	define_feature!(55, Keysend, [NodeContext],
 		"Feature flags for keysend payments.", set_keysend_optional, set_keysend_required,
 		supports_keysend, requires_keysend);
+	define_feature!(57, Trampoline, [NodeContext],
```

Comment:
Trampoline is introduced as Feature of NodeContext, but according to [BOLT#9](https://github.com/lightning/bolts/blob/086a8af2943a721b99e818c2c6d880efc5376688/09-features.md) it should of _IN9_ context, that is InitContext, NodeContext, and Bolt11InvoiceContext. Is the rest of the context planned to be introduced later?